src/ICSharpCode.SharpZipLib/Zip/ZipInputStream.cs

```diff
@@ -309,6 +319,40 @@ private void ReadDataDescriptor()
 			entry.Size = size;
 		}
 
+		/// <summary>
+		/// Complete any decryption processing and clear any cryptographic state.
+		/// </summary>
+		protected override void StopDecrypting()
+		{
+			base.StopDecrypting();
+
+			if (entry.AESKeySize != 0)
+			{
+				byte[] authBytes = new byte[ZipConstants.AESAuthCodeLength];
+				int authBytesRead = inputBuffer.ReadRawBuffer(authBytes, 0, authBytes.Length);
+
+				if (authBytesRead < ZipConstants.AESAuthCodeLength)
+				{
+					throw new Exception("Internal error missed auth code"); // Coding bug
+																			// Final block done. Check Auth code.
+				}
+
+				/*
+				byte[] calcAuthCode = (this.cryptoTransform as ZipAESTransform).GetAuthCode();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/piksel/SharpZipLib/pullRequest?prid=5745488)